cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -285,22 +285,44 @@ public static void gitClone(Path directory, String originUrl) throws IOException
 
     public static String runCommand(Path directory, String... command) throws IOException, InterruptedException {
         // Function to Run Commands using Process Builder
-        ProcessResult presult;
-        try {
-            System.out.println("Running " + String.join(" ", command));
-            presult = new ProcessExecutor().command(command).redirectOutput(System.out).redirectErrorStream(true).readOutput(true)
-                    .execute();
-        } catch (TimeoutException e) {
-            throw new RuntimeException("Error running command", e);
-        }
+         ProcessResult presult;
+
+        if (isWindows()) {
+            try{
+                System.out.println("Running " + String.join(" ", command));
+                presult = new ProcessExecutor().command("CMD", "/C", command[0], command[1]).redirectOutput(System.out).redirectErrorStream(true).readOutput(true)
```

Comment:
why are you assuming that commands array will be of max size 2. can't there be more args than that ? 
Infact there itself look at the gitClone Function which is calling run command with how many args:
` runCommand(directory.getParent(), "git", "clone", originUrl, directory.toString());`

Reply:
my mistake. its done now I think.